transportation_protocols/tcp.py

```diff
@@ -0,0 +1,35 @@
+import socket
+
+
+class TCP_Connection:
```

Comment:
Create an interface (e.g. Comm_Device) that defines send and receive.
Implement TCP_Connection and UDP_Connection as sub classes of that interface.
Read about implementing an interface in python (abs / metaclass)

Reply:
good idea
**done!**